lib/presentation/widget/upload_file/upload_file_viewmodel.dart

```diff
@@ -289,6 +308,18 @@ class UploadFileViewModel extends BaseViewModel {
     store.dispatch(MySpaceClearSelectedDocumentsAction());
   }
 
+  void _checkContactPermission() async {
+    final permissionStatus = await Permission.contacts.status;
+    if (permissionStatus.isGranted) {
+      _contactSuggestionSource = ContactSuggestionSource.all;
+    } else if (!permissionStatus.isPermanentlyDenied) {
+      final requestedPermission = await Permission.contacts.request();
+      _contactSuggestionSource = requestedPermission == PermissionStatus.granted
+          ? ContactSuggestionSource.all
+          : _contactSuggestionSource;
+    }
+  }
```

Comment:
I did not see the case for `permanentDenied`, so that, user can not use this if he makes any mistake decision